tools/flow/tasks.go

```diff
@@ -181,6 +181,9 @@ func (c *Controller) findImpliedTask(d dep.Dependency) *Task {
 
 	for ; n != nil; n = n.Parent {
 		if c.cfg.IgnoreConcrete && n.IsConcrete() {
+			if n.BaseValue == nil {
+				return nil
+			}
```

Comment:
As far as I can tell, BaseValue should never be nil at this point. So the bug is likely elsewhere.